test/images/Makefile

```diff
@@ -16,7 +16,7 @@ REGISTRY ?= gcr.io/kubernetes-e2e-test-images
 GOARM ?= 7
 DOCKER_CERT_BASE_PATH ?=
 QEMUVERSION=v2.9.1
-GOLANG_VERSION=1.15rc2
+GOLANG_VERSION=1.15.0
```

Comment:
is this just `1.15` or is it `1.15.0`?

Reply:
the docker image includes the `.0` 😜 

https://hub.docker.com/_/golang?tab=tags&page=1&name=1.15.0